### PR TITLE
Add feature_can_always_use_hypothesis parameter

### PR DESCRIPTION
**Pull request by thewilkybarkid**
For https://github.com/elifesciences/journal/pull/853.

### CODE REVIEW

File: salt/journal/config/srv-journal-local-demo-app-config-parameters.yml [hunk 1]
```diff
@@ -36,3 +36,5 @@ parameters:
     hypothesis_authority: some-authority
     hypothesis_client_id: some-id
     hypothesis_client_secret: ThisIsASecret
+
+    feature_can_always_use_hypothesis: true
```
Comment (thewilkybarkid) [inline salt/journal/config/srv-journal-local-demo-app-config-parameters.yml]: It already uses `?open-sesame`, so this value doesn't matter.

Comment (giorgiosironi) [reply salt/journal/config/srv-journal-local-demo-app-config-parameters.yml]: (as long as the bootstrap runs)

**Comment by giorgiosironi** [comment]
Creating `journal--pr-45-manual`

**Comment by giorgiosironi** [comment]
(`ssh elife@pr-45-manual--journal.elifesciences.org`)

**Comment by thewilkybarkid** [comment]
https://github.com/elifesciences/journal/blob/f6265ec335420df7a7d1d3c846992923fad6f687/gulpfile.js#L352 does work as the path is appended to it, so I think every page in production now has the homepage critical CSS. It's redundant when this is merged (though will continue to do the homepage until fixed).

**Comment by thewilkybarkid** [comment]
Just spotted that the query string doesn't work and it's our (mine I think) fault - https://github.com/elifesciences/journal/blob/f9d5c3f43a490426104c1c7b90b1ac10c7ccb192/gulpfile.js#L217 /cc @davidcmoulton Feel free to make that safer for future usage!

**Pull request merged**